lib/parser/PaginateableParser.js

```
@@ -0,0 +1,218 @@
+try {
+    var RedParser = require('./RedParser');
+} catch (e) {
```

it's really bad practice to ignore errors this way